DataDog/datadog-agent
# [process-agent] Add loopback resolution

### What does this PR do?

Add resolution for loopback to loopback flows

### Motivation

This is required for visibility within pods

### Additional Notes

### Describe your test plan
1. Setup istio on a k8s cluster
2. Deploy an nginx pod with istio injected
3. curl the pod IP of the nginx pod
4. In flow explorer, check for loopback to loopback connections, dest port 80, source port ephemeral. These should resolve to source container as the istio sidecar proxy container and the dest container as the nginx container.
